build.rs

```diff
@@ -0,0 +1,36 @@
+extern crate bindgen;
```

Comment:
Add a top-level comment explaining where the `WFA2-lib` directory should be, and how to compile that as a library.
You can use `//!` for this.